single-node-refactor/src/Solvers/SGH_solver_rz/src/force_sgh_rz.cpp

```diff
@@ -414,11 +414,21 @@ void SGHRZ::get_force_rz(const Material_t& Materials,
                 // Wilkins used elem_area*0.25 for the corner area, we will use the corner
                 // areas calculated using Barlow's symmetry and energy preserving area partitioning
                 if (node_radius > tiny) {
+                    
                     // sigma_RZ / R_p
-                    corner_force(corner_gid, 0) += tau(1, 0) * corner_areas(corner_lid) / node_radius;
+                    double force_term_1 = tau(1, 0) * corner_areas(corner_lid) / node_radius; 
```

Comment:
Can you discuss what force_term_1 and force_term_2 are here?  What caused them to be broken up like this?